docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+title: Browser SDK
+description: Ship browser metrics to Sematext Experience.
+
+Sematext [Browser SDK](https://github.com/sematext/browser-sdk) is an open-source framework used to ship web application metrics related to user interaction. Using [Browser SDK](https://github.com/sematext/browser-sdk) allows gathering metrics related to:
+ 
+ - Hard page loads for traditional web applications as well as the single-page applications
+ - Soft page loads generated via the `routeChange` command for the single-page applications
+ - HTTP requests
+ - [Web application resources, such as images, CSS files, etc.](/experience/resources/)
+ - [Web vitals](/experience/webvitals/)
+ - [Element timing](/experience/element-timing/)
+ - [Long tasks](/experience/longtasks/)
+ - [User sessions](/experience/user-identification/)
+ - [On-page transactions](/experience/on-page-transaction/)
+
+![Sematext Experience Overview](../images/experience/overview.png)
+
+## Development
+
+The RUM script is developed using [ECMAScript 2015](https://en.wikipedia.org/wiki/ECMAScript) and uses a few tools to help with the build and working with the code. Those tools include:
```

Comment:
> The RUM script is developed using [ECMAScript 2015](https://en.wikipedia.org/wiki/ECMAScript) and uses

The RUM script is developed using [ECMAScript 2015](https://en.wikipedia.org/wiki/ECMAScript).  For builds and development it uses:

Reply:
Done.